### DISCUSSION TITLE
Table with expanding rows problem

### DISCUSSION THREAD

**Discussion by maicss** [quasarframework/quasar#12898] [General - Components / Directives / etc]
How to auto wrap very long content?

https://codepen.io/maicss/pen/jOYVryX?editors=1010

Simple text also display this

**Comment by corepay** [comment]
I have same problem.  What I did to fix the table row from stretching out the columns if my expansion row data was real wide was set a max-width on the data in the expansion row like this: 

```
<q-tr v-show="props.expand" :props="props" >
    <q-td colspan="100%">
        <div v-html="props.row.markup" style="max-width:250px"></div>
    </q-td>
</q-tr>
```

In your codepen I wrapped your q-chips in a div with a max-width and got this:

![image](https://user-images.githubusercontent.com/48992923/159929311-1fb5118c-1f0f-4956-adf1-b5e00e69e2e9.png)

**Reply by corepay** [reply]
You can also calculate your row width that you desire and set as max-width with 'overflow:auto' to add scroll bars on your expansion row if you need to.

**Reply by maicss** [reply]
`max-width` is not working, `max-width` with `overflow` shows a horizontal scrollbar, it's ugly...
